asciidoc/workloads/azure/backend/java/setting_up_cosmos_db_locally_java.adoc

```diff
@@ -15,10 +15,15 @@ Latest instruction can be found in https://docs.microsoft.com/en-us/azure/cosmos
 Download and install the latest version of **CosmosDB Emulator**. Once it's installed, it will be
 available on https://localhost:8081/_explorer/index.html[localhost].
 
-=== Windows
+.Windows
+[%collapsible]
+====
 You can also find the Emulator from Start Menu and type Azure CosmosDB Emulator.
+====
 
-=== Unix
+.Unix
+[%collapsible]
+====
```

Comment:
This collapsible section should only contain the proceeding paragraph which ends with the ip 127.0.0.1

The rest applies to both OS's

Reply:
good spot